modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItReplicaLifecycleTest.java

```diff
@@ -984,7 +1018,8 @@ class Node {
 
             LogicalTopologyServiceImpl logicalTopologyService = new LogicalTopologyServiceImpl(logicalTopology, cmgManager);
 
-            KeyValueStorage keyValueStorage = new SimpleInMemoryKeyValueStorage(name);
+            KeyValueStorage keyValueStorage =
```

Comment:
Why do we need real RocksDB?

Reply:
Because the real SimpleInMemoryKeyValueStorage produce null values under the pressure of default zone rebalances. I will create the issue about it and separate reproduce